user/admin.py

```diff
@@ -1,5 +1,35 @@
 from django.contrib import admin
-from django.contrib.auth.admin import UserAdmin
+from django.contrib.auth.admin import UserAdmin as BaseUserAdmin
+from django.utils.translation import gettext_lazy as _
 from .models import User
 
+
+class UserAdmin(BaseUserAdmin):
+    ordering = ["email"]
```

Comment:
The `ordering` attribute is set to `email`, which assumes that the `User` model has an `email` field. Ensure that the `User` model indeed has an `email` field, as this is not a default field in Django's `AbstractUser` model.